_data/students.yml

```diff
@@ -348,3 +348,9 @@
   facebook: maitreyav
   twitter: maitvyas
   bio: 17, Second GCI, Design + Code, The web is what you make of it. 
+
+- name: Catalin Pirvu
+  github: https://github.com/CatalinPirvu
+  image: catalinpirvu.jpg
+  facebook: https://www.facebook.com/pirvu.catalin.98
```

Comment:
Hey @CatalinPirvu , You just need to add user ids in Facebook and GitHub. Follow the pattern, no need of complete URLs.